test/auth.e2e-spec.ts

```diff
@@ -2,85 +2,141 @@ import { INestApplication } from '@nestjs/common';
+    await roleRepo.save(role);
+    await appPlanRepo.save(appPlan);
+    await appRepo.save(app);
+    await memberRepo.save(member);
+    await application.init();
   });
 
-  describe('/auth/token (POST)', () => {
-    const route = '/auth/token';
-
-    it('Should return E_NOT_FOUND error', async () => {
-      const { body } = await request(app.getHttpServer())
-        .post(route)
-        .send({
-          clientId: 'not_exists',
-          key: 'not_exists',
-          permissions: [],
-        })
-        .expect(400);
-      
+  afterEach(async () => {
+    await authAutditLogRepo.delete({});
+    await memberRepo.delete({});
+    await appRepo.delete({});
+    await appPlanRepo.delete({});
+    await roleRepo.delete({});
+
+    await application.close();
+  });
+
+  describe('/auth/password/temporary (POST)', () => {
+    const route = '/auth/password/temporary';
+
+    it('should return E_NO_MEMBER error', async () => {
+      const { body } = await request(application.getHttpServer()).post(route).send({
+        appId: 'test',
+        applicant: member.id,
+        email: 'test_nonexist@example.com',
+        purpose: 'test',
+      });
+
       expect(body).toStrictEqual({
-        code: 'E_NOT_FOUND',
-        message: 'client ID doesn\'t exist',
+        code: 'E_NO_MEMBER',
+        message: 'member not found',
         result: null,
       });
     });
 
-    // TODO:
-    //   After Hasura containerize, build test set and uncomment code below.
-    // it('Should return E_AUTH_TOKEN error', async () => {
-    //   const { body } = await request(app.getHttpServer())
-    //     .post('/auth/token')
-    //     .send({
-    //       clientId: 'not_exists',
-    //       key: 'not_exists',
-    //       permissions: [],
-    //     })
-    //     .expect(400);
-      
-    //   expect(body).toStrictEqual({
-    //     code: 'E_AUTH_TOKEN',
-    //     message: 'key is not authenticated',
-    //     result: null,
-    //   });
-    // });
-
-    // it('Should return authToken', async () => {
-    //   const { body } = await request(app.getHttpServer())
-    //     .post('/auth/token')
-    //     .send({
-    //       clientId: 'not_exists',
-    //       key: 'not_exists',
-    //       permissions: [],
-    //     })
-    //     .expect(200);
-      
-    //   const { authToken } = body;
-    //   expect(body).toStrictEqual({
-    //     code: 'SUCCESS',
-    //     message: 'get auth token successfully',
-    //     result: { authToken },
-    //   });
-    // });
+    it('should successfully retrieve a temporary password', async () => {
+      const { body } = await request(application.getHttpServer()).post(route).send({
+        appId: member.appId,
+        applicant: member.id,
+        email: member.email,
+        purpose: 'test',
+      });
+
+      expect(body.code).toBe('SUCCESS');
+      expect(body.result.password).toBeDefined();
+    });
+
+    it('should fail to retrieve a temporary password', async () => {
```

Comment:
'should fail to retrieve a temporary password'
-> Describe with situation
e.g.: 'Should fail due to ......'